azure/services/scalesets/scalesets.go

```diff
@@ -452,6 +453,15 @@ func (s *Service) buildVMSSFromSpec(ctx context.Context, vmssSpec azure.ScaleSet
 	})
 
 	vmss.Tags = converters.TagsToMap(tagsWithoutHash)
+
+	if vmss.VirtualMachineProfile.OsProfile.CustomData != nil {
+		cloneCustomData := *vmss.VirtualMachineProfile.OsProfile.CustomData
+		vmss.VirtualMachineProfile.OsProfile.CustomData = nil
+		defer func() {
+			vmss.VirtualMachineProfile.OsProfile.CustomData = to.StringPtr(cloneCustomData)
+		}()
+	}
```

Comment:
should this be in same place as the exclusion for the windows pass?

Reply:
Yes, indeed. Moving it there. Thank you.